packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -120,33 +105,24 @@ Chunking is relevant for both `FlushMode.TurnBased` and `FlushMode.Immediate` as
 
 ## Disabling in case of emergency
 
-If the features are enabled using the configs, they can be disabled at runtime via feature gates as following:
+Compression and Chunking configuration can be overridden via feature gates to force-disable them:
 
 -   `Fluid.ContainerRuntime.CompressionDisabled` - if set to true, will disable compression (this has a side effect of also disabling chunking, as chunking is invoked only for compressed payloads).
--   `Fluid.ContainerRuntime.DisableGroupedBatching` - if set to true, will disable grouped batching.
```

Comment:
@kian-thompson , @vladsud -- We recently removed this feature gate, can we do the same for compression and chunking?

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/8122